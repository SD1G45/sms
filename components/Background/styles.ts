import styled from "styled-components";

export const FirstColumn = styled.div`
  height: 100vh;
  width: 260px;
`;
export const Column = styled.div`
  height: 100vh;

  width: 250px;
  border-left: 1px dashed #ccc;
`;

export const TiltedDiv = styled.div`
  height: 100vh;
  width: 230vw;
  position: absolute;
  bottom: 80vh;
  left: -60vw;

  background-color: #f7f8ff;
  transform: rotate(165deg);
  z-index: -2;
`;
export const StyledBackground = styled.div`
  position: absolute;
  display: flex;
  justify-content: space-between;
  width: 100%;
  z-index: -1;
`;
