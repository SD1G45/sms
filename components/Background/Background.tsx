import React from "react";
import { StyledBackground, FirstColumn, Column, TiltedDiv } from "./styles";

const Background = () => {
  return (
    <StyledBackground>
      <TiltedDiv />
      <FirstColumn />
      <Column />
      <Column />
      <Column />
      <Column />
    </StyledBackground>
  );
};

export default Background;
