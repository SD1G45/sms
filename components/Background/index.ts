import { Background } from "../../page-styles/login/styles";

export { default } from "./Background";
