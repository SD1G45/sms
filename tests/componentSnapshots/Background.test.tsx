import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Background from "../../components/Background";

test("Background renders correctly", () => {
  const tree = renderer.create(<Background/>).toJSON();
  expect(tree).toMatchSnapshot();
});