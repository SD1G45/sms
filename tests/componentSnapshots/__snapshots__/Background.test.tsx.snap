// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background renders correctly 1`] = `
.c2 {
  height: 100vh;
  width: 260px;
}

.c3 {
  height: 100vh;
  width: 250px;
  border-left: 1px dashed #ccc;
}

.c1 {
  height: 100vh;
  width: 230vw;
  position: absolute;
  bottom: 55vh;
  left: -60vw;
  background-color: #f7f8ff;
  -webkit-transform: rotate(165deg);
  -ms-transform: rotate(165deg);
  transform: rotate(165deg);
  z-index: -2;
}

.c0 {
  position: absolute;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 100%;
  z-index: -1;
}

<div
  className="c0"
>
  <div
    className="c1"
  />
  <div
    className="c2"
  />
  <div
    className="c3"
  />
  <div
    className="c3"
  />
  <div
    className="c3"
  />
  <div
    className="c3"
  />
  <div
    className="c3"
  />
  <div
    className="c3"
  />
  <div
    className="c3"
  />
</div>
`;
